Can't start process after update to 3.6.3

**Describe the bug**
After updating to from 3.6.2 to 3.6.3 I can't start an app with npm run dev.

**To Reproduce**
Steps to reproduce the behavior:

1. Update all packages from 3.6.2 to 3.6.3
2. Run npm run dev
3. See error

**Expected behavior**
App starts normally

**Actual behavior**
App does not start

**Screenshots/Videos**

**Error logs**

```
> vendure@1.0.0 dev
> concurrently --kill-others npm:dev:*

[dashboard] 
[dashboard] > vendure@1.0.0 dev:dashboard
[dashboard] > vite --clearScreen false
[dashboard] 
[server] 
[server] > vendure@1.0.0 dev:server
[server] > ts-node ./src/index.ts
[server] 
[worker] 
[worker] > vendure@1.0.0 dev:worker
[worker] > ts-node ./src/index-worker.ts
[worker] 
[dashboard] 11:51:05 AM [vite] (client) info: Loading Vendure config. This can take a short while depending on the size of your project...
[dashboard]   Plugin: vendure:config-loader
[dashboard] 11:51:05 AM [vite] (client) info: TypeScript compilation completed in 167ms
[dashboard]   Plugin: vendure:config-loader
[dashboard] 11:51:05 AM [vite] (client) info: Found 82 potential plugin files in 168ms (scanned 2989 files)
[dashboard]   Plugin: vendure:config-loader
[dashboard] 11:51:05 AM [vite] (client) info: Analyzed plugins and found 1 dashboard extensions in 290ms
[dashboard]   Plugin: vendure:config-loader
[dashboard] 11:51:07 AM [vite] (client) info: Found 1 plugins: ProductVariantExtensionPlugin (local)
[dashboard]   Plugin: vendure:config-loader
[dashboard] 11:51:07 AM [vite] (client) info: Vendure config loaded (using export "config") in 2094ms
[dashboard]   Plugin: vendure:config-loader
[dashboard] error when starting dev server:
[dashboard] TypeError: Received invalid input.{ __validationErrors: undefined, description: undefined, extensions: {}, astNode: { kind: "SchemaDefinition", description: undefined, directives: [], operationTypes: [[Object], [Object]], loc: [Object] }, extensionASTNodes: [], _queryType: Query, _mutationType: Mutation, _subscriptionType: undefined, _directives: [@include, @skip, @deprecated, @specifiedBy, @oneOf], _typeMap: { Query: Query, ID: ID, String: String, Int: Int, Mutation: Mutation, Boolean: Boolean, AdministratorListOptions: AdministratorListOptions, CreateAdministratorInput: CreateAdministratorInput, UpdateAdministratorInput: UpdateAdministratorInput, UpdateActiveAdministratorInput: UpdateActiveAdministratorInput, Administrator: Administrator, AdministratorList: AdministratorList, CreateApiKeyTranslationInput: CreateApiKeyTranslationInput, UpdateApiKeyTranslationInput: UpdateApiKeyTranslationInput, CreateApiKeyInput: CreateApiKeyInput, CreateApiKeyResult: CreateApiKeyResult, UpdateApiKeyInput: UpdateApiKeyInput, RotateApiKeyResult: RotateApiKeyResult, ApiKey: ApiKey, ApiKeyTranslation: ApiKeyTranslation, ApiKeyListOptions: ApiKeyListOptions, ApiKeyList: ApiKeyList, MimeTypeError: MimeTypeError, CreateAssetResult: CreateAssetResult, AssetListOptions: AssetListOptions, AssetTranslationInput: AssetTranslationInput, CreateAssetInput: CreateAssetInput, CoordinateInput: CoordinateInput, Float: Float, DeleteAssetInput: DeleteAssetInput, DeleteAssetsInput: DeleteAssetsInput, UpdateAssetInput: UpdateAssetInput, AssignAssetsToChannelInput: AssignAssetsToChannelInput, AuthenticationInput: AuthenticationInput, NativeAuthenticationResult: NativeAuthenticationResult, AuthenticationResult: AuthenticationResult, ChannelList: ChannelList, ChannelListOptions: ChannelListOptions, CreateChannelInput: CreateChannelInput, UpdateChannelInput: UpdateChannelInput, LanguageNotAvailableError: LanguageNotAvailableError, CreateChannelResult: CreateChannelResult, UpdateChannelResult: UpdateChannelResult, Collection: Collection, CollectionListOptions: CollectionListOptions, MoveCollectionInput: MoveCollectionInput, CreateCollectionTranslationInput: CreateCollectionTranslationInput, UpdateCollectionTranslationInput: UpdateCollectionTranslationInput, CreateCollectionInput: CreateCollectionInput, PreviewCollectionVariantsInput: PreviewCollectionVariantsInput, UpdateCollectionInput: UpdateCollectionInput, AssignCollectionsToChannelInput: AssignCollectionsToChannelInput, RemoveCollectionsFromChannelInput: RemoveCollectionsFromChannelInput, CountryTranslationInput: CountryTranslationInput, CreateCountryInput: CreateCountryInput, UpdateCountryInput: UpdateCountryInput, CountryListOptions: CountryListOptions, Customer: Customer, CustomerGroupList: CustomerGroupList, CustomerGroupListOptions: CustomerGroupListOptions, CreateCustomerGroupInput: CreateCustomerGroupInput, UpdateCustomerGroupInput: UpdateCustomerGroupInput, UpdateCustomerInput: UpdateCustomerInput, CustomerFilterParameter: CustomerFilterParameter, CustomerListOptions: CustomerListOptions, AddNoteToCustomerInput: AddNoteToCustomerInput, UpdateCustomerNoteInput: UpdateCustomerNoteInput, CreateCustomerResult: CreateCustomerResult, UpdateCustomerResult: UpdateCustomerResult, EntityDuplicatorDefinition: EntityDuplicatorDefinition, DuplicateEntitySuccess: DuplicateEntitySuccess, DuplicateEntityError: DuplicateEntityError, DuplicateEntityResult: DuplicateEntityResult, DuplicateEntityInput: DuplicateEntityInput, Facet: Facet, FacetListOptions: FacetListOptions, FacetTranslationInput: FacetTranslationInput, CreateFacetInput: CreateFacetInput, UpdateFacetInput: UpdateFacetInput, FacetValueTranslationInput: FacetValueTranslationInput, CreateFacetValueWithFacetInput: CreateFacetValueWithFacetInput, CreateFacetValueInput: CreateFacetValueInput, UpdateFacetValueInput: UpdateFacetValueInput, AssignFacetsToChannelInput: AssignFacetsToChannelInput, RemoveFacetsFromChannelInput: RemoveFacetsFromChannelInput, FacetInUseError: FacetInUseError, RemoveFacetFromChannelResult: RemoveFacetFromChannelResult, UpdateGlobalSettingsInput: UpdateGlobalSettingsInput, ChannelDefaultLanguageError: ChannelDefaultLanguageError, UpdateGlobalSettingsResult: UpdateGlobalSettingsResult, GlobalSettings: GlobalSettings, OrderProcessState: OrderProcessState, PermissionDefinition: PermissionDefinition, ServerConfig: ServerConfig, HistoryEntry: HistoryEntry, ImportInfo: ImportInfo, JobBufferSize: JobBufferSize, JobState: JobState, JobListOptions: JobListOptions, JobList: JobList, Job: Job, JobQueue: JobQueue, Order: Order, Fulfillment: Fulfillment, Payment: Payment, OrderModificationLine: OrderModificationLine, OrderModification: OrderModification, OrderFilterParameter: OrderFilterParameter, OrderSortParameter: OrderSortParameter, OrderListOptions: OrderListOptions, SetOrderCustomerInput: SetOrderCustomerInput, UpdateOrderInput: UpdateOrderInput, FulfillOrderInput: FulfillOrderInput, CancelOrderInput: CancelOrderInput, RefundOrderInput: RefundOrderInput, OrderLineInput: OrderLineInput, SettleRefundInput: SettleRefundInput, AddNoteToOrderInput: AddNoteToOrderInput, UpdateOrderNoteInput: UpdateOrderNoteInput, AdministratorPaymentInput: AdministratorPaymentInput, AdministratorRefundInput: AdministratorRefundInput, ModifyOrderOptions: ModifyOrderOptions, UpdateOrderAddressInput: UpdateOrderAddressInput, ModifyOrderInput: ModifyOrderInput, AddItemInput: AddItemInput, SurchargeInput: SurchargeInput, ManualPaymentInput: ManualPaymentInput, AddItemToDraftOrderInput: AddItemToDraftOrderInput, AdjustDraftOrderLineInput: AdjustDraftOrderLineInput, SettlePaymentError: SettlePaymentError, CancelPaymentError: CancelPaymentError, EmptyOrderLineSelectionError: EmptyOrderLineSelectionError, ItemsAlreadyFulfilledError: ItemsAlreadyFulfilledError, InvalidFulfillmentHandlerError: InvalidFulfillmentHandlerError, CreateFulfillmentError: CreateFulfillmentError, InsufficientStockOnHandError: InsufficientStockOnHandError, MultipleOrderError: MultipleOrderError, CancelActiveOrderError: CancelActiveOrderError, PaymentOrderMismatchError: PaymentOrderMismatchError, RefundOrderStateError: RefundOrderStateError, NothingToRefundError: NothingToRefundError, AlreadyRefundedError: AlreadyRefundedError, QuantityTooGreatError: QuantityTooGreatError, RefundAmountError: RefundAmountError, RefundStateTransitionError: RefundStateTransitionError, PaymentStateTransitionError: PaymentStateTransitionError, FulfillmentStateTransitionError: FulfillmentStateTransitionError, OrderModificationStateError: OrderModificationStateError, NoChangesSpecifiedError: NoChangesSpecifiedError, PaymentMethodMissingError: PaymentMethodMissingError, RefundPaymentIdMissingError: RefundPaymentIdMissingError, ManualPaymentStateError: ManualPaymentStateError, TransitionOrderToStateResult: TransitionOrderToStateResult, SettlePaymentResult: SettlePaymentResult, CancelPaymentResult: CancelPaymentResult, AddFulfillmentToOrderResult: AddFulfillmentToOrderResult, CancelOrderResult: CancelOrderResult, RefundOrderResult: RefundOrderResult, SettleRefundResult: SettleRefundResult, TransitionFulfillmentToStateResult: TransitionFulfillmentToStateResult, TransitionPaymentToStateResult: TransitionPaymentToStateResult, ModifyOrderResult: ModifyOrderResult, AddManualPaymentToOrderResult: AddManualPaymentToOrderResult, SetCustomerForDraftOrderResult: SetCustomerForDraftOrderResult, PaymentMethodList: PaymentMethodList, PaymentMethodListOptions: PaymentMethodListOptions, PaymentMethodTranslationInput: PaymentMethodTranslationInput, CreatePaymentMethodInput: CreatePaymentMethodInput, UpdatePaymentMethodInput: UpdatePaymentMethodInput, AssignPaymentMethodsToChannelInput: AssignPaymentMethodsToChannelInput, RemovePaymentMethodsFromChannelInput: RemovePaymentMethodsFromChannelInput, Product: Product, ProductVariantPrice: ProductVariantPrice, ProductVariant: ProductVariant, ProductOptionGroup: ProductOptionGroup, ProductOptionGroupListOptions: ProductOptionGroupListOptions, ProductOptionListOptions: ProductOptionListOptions, ProductOptionGroupList: ProductOptionGroupList, ProductOptionList: ProductOptionList, ProductOptionGroupTranslationInput: ProductOptionGroupTranslationInput, CreateProductOptionGroupInput: CreateProductOptionGroupInput, UpdateProductOptionGroupInput: UpdateProductOptionGroupInput, ProductOptionTranslationInput: ProductOptionTranslationInput, CreateGroupOptionInput: CreateGroupOptionInput, CreateProductOptionInput: CreateProductOptionInput, UpdateProductOptionInput: UpdateProductOptionInput, AssignProductOptionGroupsToChannelInput: AssignProductOptionGroupsToChannelInput, RemoveProductOptionGroupsFromChannelInput: RemoveProductOptionGroupsFromChannelInput, ProductOptionGroupInUseError: ProductOptionGroupInUseError, RemoveProductOptionGroupFromChannelResult: RemoveProductOptionGroupFromChannelResult, SearchResult: SearchResult, StockMovementListOptions: StockMovementListOptions, ProductListOptions: ProductListOptions, ProductFilterParameter: ProductFilterParameter, ProductVariantListOptions: ProductVariantListOptions, ProductVariantFilterParameter: ProductVariantFilterParameter, ProductTranslationInput: ProductTranslationInput, CreateProductInput: CreateProductInput, UpdateProductInput: UpdateProductInput, ProductVariantTranslationInput: ProductVariantTranslationInput, CreateProductVariantOptionInput: CreateProductVariantOptionInput, StockLevelInput: StockLevelInput, CreateProductVariantPriceInput: CreateProductVariantPriceInput, UpdateProductVariantPriceInput: UpdateProductVariantPriceInput, CreateProductVariantInput: CreateProductVariantInput, UpdateProductVariantInput: UpdateProductVariantInput, AssignProductsToChannelInput: AssignProductsToChannelInput, RemoveProductsFromChannelInput: RemoveProductsFromChannelInput, AssignProductVariantsToChannelInput: AssignProductVariantsToChannelInput, RemoveProductVariantsFromChannelInput: RemoveProductVariantsFromChannelInput, ProductOptionInUseError: ProductOptionInUseError, RemoveOptionGroupFromProductResult: RemoveOptionGroupFromProductResult, PromotionListOptions: PromotionListOptions, PromotionTranslationInput: PromotionTranslationInput, CreatePromotionInput: CreatePromotionInput, UpdatePromotionInput: UpdatePromotionInput, AssignPromotionsToChannelInput: AssignPromotionsToChannelInput, RemovePromotionsFromChannelInput: RemovePromotionsFromChannelInput, MissingConditionsError: MissingConditionsError, CreatePromotionResult: CreatePromotionResult, UpdatePromotionResult: UpdatePromotionResult, ProvinceTranslationInput: ProvinceTranslationInput, CreateProvinceInput: CreateProvinceInput, UpdateProvinceInput: UpdateProvinceInput, ProvinceListOptions: ProvinceListOptions, RoleListOptions: RoleListOptions, CreateRoleInput: CreateRoleInput, UpdateRoleInput: UpdateRoleInput, UpdateScheduledTaskInput: UpdateScheduledTaskInput, ScheduledTask: ScheduledTask, SellerList: SellerList, SellerListOptions: SellerListOptions, CreateSellerInput: CreateSellerInput, UpdateSellerInput: UpdateSellerInput, SettingsStoreScopeType: SettingsStoreScopeType, SettingsStoreFieldDefinition: SettingsStoreFieldDefinition, SettingsStoreInput: SettingsStoreInput, SetSettingsStoreValueResult: SetSettingsStoreValueResult, ShippingMethodListOptions: ShippingMethodListOptions, ShippingMethodTranslationInput: ShippingMethodTranslationInput, CreateShippingMethodInput: CreateShippingMethodInput, UpdateShippingMethodInput: UpdateShippingMethodInput, TestShippingMethodInput: TestShippingMethodInput, TestEligibleShippingMethodsInput: TestEligibleShippingMethodsInput, TestShippingMethodOrderLineInput: TestShippingMethodOrderLineInput, TestShippingMethodResult: TestShippingMethodResult, TestShippingMethodQuote: TestShippingMethodQuote, AssignShippingMethodsToChannelInput: AssignShippingMethodsToChannelInput, RemoveShippingMethodsFromChannelInput: RemoveShippingMethodsFromChannelInput, SlugForEntityInput: SlugForEntityInput, StockLevel: StockLevel, StockLocationListOptions: StockLocationListOptions, StockLocationList: StockLocationList, CreateStockLocationInput: CreateStockLocationInput, UpdateStockLocationInput: UpdateStockLocationInput, DeleteStockLocationInput: DeleteStockLocationInput, AssignStockLocationsToChannelInput: AssignStockLocationsToChannelInput, RemoveStockLocationsFromChannelInput: RemoveStockLocationsFromChannelInput, StockLocation: StockLocation, StockMovementType: StockMovementType, StockMovement: StockMovement, StockAdjustment: StockAdjustment, Allocation: Allocation, Sale: Sale, Cancellation: Cancellation, Return: Return, Release: Release, StockMovementItem: StockMovementItem, StockMovementList: StockMovementList, TagListOptions: TagListOptions, CreateTagInput: CreateTagInput, UpdateTagInput: UpdateTagInput, TaxCategoryList: TaxCategoryList, TaxCategoryListOptions: TaxCategoryListOptions, CreateTaxCategoryInput: CreateTaxCategoryInput, UpdateTaxCategoryInput: UpdateTaxCategoryInput, TaxRateFilterParameter: TaxRateFilterParameter, TaxRateListOptions: TaxRateListOptions, CreateTaxRateInput: CreateTaxRateInput, UpdateTaxRateInput: UpdateTaxRateInput, ZoneList: ZoneList, ZoneListOptions: ZoneListOptions, CreateZoneInput: CreateZoneInput, UpdateZoneInput: UpdateZoneInput, Address: Address, Asset: Asset, AssetTranslation: AssetTranslation, Coordinate: Coordinate, AssetList: AssetList, AssetType: AssetType, CurrentUser: CurrentUser, CurrentUserChannel: CurrentUserChannel, Channel: Channel, CollectionBreadcrumb: CollectionBreadcrumb, CollectionTranslation: CollectionTranslation, CollectionList: CollectionList, GlobalFlag: GlobalFlag, AdjustmentType: AdjustmentType, DeletionResult: DeletionResult, Permission: Permission, SortOrder: SortOrder, ErrorCode: ErrorCode, LogicalOperator: LogicalOperator, NativeAuthStrategyError: NativeAuthStrategyError, InvalidCredentialsError: InvalidCredentialsError, OrderStateTransitionError: OrderStateTransitionError, EmailAddressConflictError: EmailAddressConflictError, GuestCheckoutError: GuestCheckoutError, OrderLimitError: OrderLimitError, NegativeQuantityError: NegativeQuantityError, InsufficientStockError: InsufficientStockError, CouponCodeInvalidError: CouponCodeInvalidError, CouponCodeExpiredError: CouponCodeExpiredError, CouponCodeLimitError: CouponCodeLimitError, OrderModificationError: OrderModificationError, IneligibleShippingMethodError: IneligibleShippingMethodError, NoActiveOrderError: NoActiveOrderError, OrderInterceptorError: OrderInterceptorError, UpdateOrderItemErrorResult: UpdateOrderItemErrorResult, JSON: JSON, DateTime: DateTime, Upload: Upload, Money: Money, PaginatedList: PaginatedList, Node: Node, ErrorResult: ErrorResult, Adjustment: Adjustment, TaxLine: TaxLine, ConfigArg: ConfigArg, ConfigArgDefinition: ConfigArgDefinition, ConfigurableOperation: ConfigurableOperation, ConfigurableOperationDefinition: ConfigurableOperationDefinition, DeletionResponse: DeletionResponse, ConfigArgInput: ConfigArgInput, ConfigurableOperationInput: ConfigurableOperationInput, StringOperators: StringOperators, IDOperators: IDOperators, BooleanOperators: BooleanOperators, NumberRange: NumberRange, NumberOperators: NumberOperators, DateRange: DateRange, DateOperators: DateOperators, StringListOperators: StringListOperators, NumberListOperators: NumberListOperators, BooleanListOperators: BooleanListOperators, IDListOperators: IDListOperators, DateListOperators: DateListOperators, FacetValueFilterInput: FacetValueFilterInput, SearchInput: SearchInput, SearchResultSortParameter: SearchResultSortParameter, CreateCustomerInput: CreateCustomerInput, CreateAddressInput: CreateAddressInput, UpdateAddressInput: UpdateAddressInput, Success: Success, ShippingMethodQuote: ShippingMethodQuote, PaymentMethodQuote: PaymentMethodQuote, UpdateOrderItemsResult: UpdateOrderItemsResult, RemoveOrderItemsResult: RemoveOrderItemsResult, SetOrderShippingMethodResult: SetOrderShippingMethodResult, ApplyCouponCodeResult: ApplyCouponCodeResult, CurrencyCode: CurrencyCode, CustomField: CustomField, StringCustomFieldConfig: StringCustomFieldConfig, StringFieldOption: StringFieldOption, LocaleStringCustomFieldConfig: LocaleStringCustomFieldConfig, IntCustomFieldConfig: IntCustomFieldConfig, FloatCustomFieldConfig: FloatCustomFieldConfig, BooleanCustomFieldConfig: BooleanCustomFieldConfig, DateTimeCustomFieldConfig: DateTimeCustomFieldConfig, RelationCustomFieldConfig: RelationCustomFieldConfig, TextCustomFieldConfig: TextCustomFieldConfig, LocaleTextCustomFieldConfig: LocaleTextCustomFieldConfig, StructField: StructField, StringStructFieldConfig: StringStructFieldConfig, IntStructFieldConfig: IntStructFieldConfig, FloatStructFieldConfig: FloatStructFieldConfig, BooleanStructFieldConfig: BooleanStructFieldConfig, DateTimeStructFieldConfig: DateTimeStructFieldConfig, TextStructFieldConfig: TextStructFieldConfig, StructFieldConfig: StructFieldConfig, StructCustomFieldConfig: StructCustomFieldConfig, LocalizedString: LocalizedString, CustomFieldConfig: CustomFieldConfig, CustomerGroup: CustomerGroup, CustomerList: CustomerList, FacetValue: FacetValue, FacetValueTranslation: FacetValueTranslation, FacetTranslation: FacetTranslation, FacetList: FacetList, FacetValueListOptions: FacetValueListOptions, FacetValueList: FacetValueList, HistoryEntryType: HistoryEntryType, HistoryEntryList: HistoryEntryList, HistoryEntryListOptions: HistoryEntryListOptions, LanguageCode: LanguageCode, OrderType: OrderType, OrderTaxSummary: OrderTaxSummary, OrderAddress: OrderAddress, OrderList: OrderList, ShippingLine: ShippingLine, Discount: Discount, OrderLine: OrderLine, RefundLine: RefundLine, Refund: Refund, FulfillmentLine: FulfillmentLine, Surcharge: Surcharge, PaymentMethod: PaymentMethod, PaymentMethodTranslation: PaymentMethodTranslation, ProductOptionGroupTranslation: ProductOptionGroupTranslation, ProductOption: ProductOption, ProductOptionTranslation: ProductOptionTranslation, SearchReindexResponse: SearchReindexResponse, SearchResponse: SearchResponse, FacetValueResult: FacetValueResult, CollectionResult: CollectionResult, SearchResultAsset: SearchResultAsset, SearchResultPrice: SearchResultPrice, SinglePrice: SinglePrice, PriceRange: PriceRange, ProductTranslation: ProductTranslation, ProductList: ProductList, ProductVariantList: ProductVariantList, ProductVariantTranslation: ProductVariantTranslation, Promotion: Promotion, PromotionTranslation: PromotionTranslation, PromotionList: PromotionList, Region: Region, RegionTranslation: RegionTranslation, Country: Country, CountryList: CountryList, Province: Province, ProvinceList: ProvinceList, Role: Role, RoleList: RoleList, Seller: Seller, ShippingMethod: ShippingMethod, ShippingMethodTranslation: ShippingMethodTranslation, ShippingMethodList: ShippingMethodList, Tag: Tag, TagList: TagList, TaxCategory: TaxCategory, TaxRate: TaxRate, TaxRateList: TaxRateList, User: User, AuthenticationMethod: AuthenticationMethod, Zone: Zone, __Schema: __Schema, __Type: __Type, __TypeKind: __TypeKind, __Field: __Field, __InputValue: __InputValue, __EnumValue: __EnumValue, __Directive: __Directive, __DirectiveLocation: __DirectiveLocation, DashboardMetricSummary: DashboardMetricSummary, DashboardMetricType: DashboardMetricType, DashboardMetricSummaryEntry: DashboardMetricSummaryEntry, DashboardMetricSummaryInput: DashboardMetricSummaryInput, ProductVariantQuality: ProductVariantQuality, Color: Color, ColorList: ColorList, ColorListOptions: ColorListOptions, UpdateColorInput: UpdateColorInput }, _subTypeMap: {}, _implementationsMap: { Node: { objects: [Array], interfaces: [Array] }, PaginatedList: { objects: [Array], interfaces: [] }, ErrorResult: { objects: [Array], interfaces: [] }, StockMovement: { objects: [Array], interfaces: [] }, CustomField: { objects: [Array], interfaces: [] }, StructField: { objects: [Array], interfaces: [] }, Region: { objects: [Array], interfaces: [] } } }
[dashboard]     at applySubschemaConfigTransforms (/node_modules/@graphql-tools/stitch/dist/index.cjs:3068:11)
[dashboard]     at stitchSchemas (/node_modules/@graphql-tools/stitch/dist/index.cjs:2980:46)
[dashboard]     at generateListOptions (/node_modules/@vendure/core/dist/api/config/generate-list-options.js:69:12)
[dashboard]     at buildSchemaFromVendureConfig (/node_modules/@vendure/core/dist/api/config/get-final-vendure-schema.js:41:62)
[dashboard]     at getFinalVendureSchema (/node_modules/@vendure/core/dist/api/config/get-final-vendure-schema.js:27:14)
[dashboard]     at async file:///node_modules/@vendure/dashboard/dist/vite/utils/schema-generator.js:19:37
[dashboard] npm run dev:dashboard exited with code 1
--> Sending SIGTERM to other processes..
[worker] /node_modules/ts-node/src/index.ts:859
[worker]     return new TSError(diagnosticText, diagnosticCodes, diagnostics);
[worker]            ^
[worker] TSError: ⨯ Unable to compile TypeScript:
[worker] src/plugins/product-variant-extension/product-variant-extension.plugin.ts(20,26): error TS2322: Type 'DocumentNode' is not assignable to type 'DocumentNode | ((schema?: GraphQLSchema | undefined) => DocumentNode | undefined) | undefined'.
[worker]   Type 'import("/node_modules/graphql/language/ast").DocumentNode' is not assignable to type 'import("/node_modules/@vendure/core/node_modules/graphql/language/ast").DocumentNode'.
[worker]     Types of property 'kind' are incompatible.
[worker]       Type 'import("/node_modules/graphql/language/kinds").Kind.DOCUMENT' is not assignable to type 'import("/node_modules/@vendure/core/node_modules/graphql/language/kinds").Kind.DOCUMENT'.
[worker]         Property 'TYPE_COORDINATE' is missing in type 'import("/node_modules/@vendure/core/node_modules/graphql/language/kinds").Kind'.
[worker] src/plugins/product-variant-extension/product-variant-extension.plugin.ts(21,27): error TS2322: Type 'DocumentNode' is not assignable to type 'DocumentNode | ((schema?: GraphQLSchema | undefined) => DocumentNode | undefined) | undefined'.
[worker]   Type 'import("/node_modules/graphql/language/ast").DocumentNode' is not assignable to type 'import("/node_modules/@vendure/core/node_modules/graphql/language/ast").DocumentNode'.
[worker]     Types of property 'kind' are incompatible.
[worker]       Type 'import("/node_modules/graphql/language/kinds").Kind.DOCUMENT' is not assignable to type 'import("/node_modules/@vendure/core/node_modules/graphql/language/kinds").Kind.DOCUMENT'.
[worker]         Property 'TYPE_COORDINATE' is missing in type 'import("/node_modules/@vendure/core/node_modules/graphql/language/kinds").Kind'.
[worker] 
[worker]     at createTSError (/node_modules/ts-node/src/index.ts:859:12)
[worker]     at reportTSError (/node_modules/ts-node/src/index.ts:863:19)
[worker]     at getOutput (/node_modules/ts-node/src/index.ts:1077:36)
[worker]     at Object.compile (/node_modules/ts-node/src/index.ts:1433:41)
[worker]     at Module.m._compile (/node_modules/ts-node/src/index.ts:1617:30)
[worker]     at node:internal/modules/cjs/loader:1943:10
[worker]     at Object.require.extensions.<computed> [as .ts] (/node_modules/ts-node/src/index.ts:1621:12)
[worker]     at Module.load (node:internal/modules/cjs/loader:1533:32)
[worker]     at Module._load (node:internal/modules/cjs/loader:1335:12)
[worker]     at wrapModuleLoad (node:internal/modules/cjs/loader:255:19) {
[worker]   diagnosticCodes: [ 2322, 2322 ]
[worker] }
[server] /node_modules/ts-node/src/index.ts:859
[server]     return new TSError(diagnosticText, diagnosticCodes, diagnostics);
[server]            ^
[server] TSError: ⨯ Unable to compile TypeScript:
[server] src/plugins/product-variant-extension/product-variant-extension.plugin.ts(20,26): error TS2322: Type 'DocumentNode' is not assignable to type 'DocumentNode | ((schema?: GraphQLSchema | undefined) => DocumentNode | undefined) | undefined'.
[server]   Type 'import("/node_modules/graphql/language/ast").DocumentNode' is not assignable to type 'import("/node_modules/@vendure/core/node_modules/graphql/language/ast").DocumentNode'.
[server]     Types of property 'kind' are incompatible.
[server]       Type 'import("/node_modules/graphql/language/kinds").Kind.DOCUMENT' is not assignable to type 'import("/node_modules/@vendure/core/node_modules/graphql/language/kinds").Kind.DOCUMENT'.
[server]         Property 'TYPE_COORDINATE' is missing in type 'import("/node_modules/@vendure/core/node_modules/graphql/language/kinds").Kind'.
[server] src/plugins/product-variant-extension/product-variant-extension.plugin.ts(21,27): error TS2322: Type 'DocumentNode' is not assignable to type 'DocumentNode | ((schema?: GraphQLSchema | undefined) => DocumentNode | undefined) | undefined'.
[server]   Type 'import("/node_modules/graphql/language/ast").DocumentNode' is not assignable to type 'import("/node_modules/@vendure/core/node_modules/graphql/language/ast").DocumentNode'.
[server]     Types of property 'kind' are incompatible.
[server]       Type 'import("/node_modules/graphql/language/kinds").Kind.DOCUMENT' is not assignable to type 'import("/node_modules/@vendure/core/node_modules/graphql/language/kinds").Kind.DOCUMENT'.
[server]         Property 'TYPE_COORDINATE' is missing in type 'import("/node_modules/@vendure/core/node_modules/graphql/language/kinds").Kind'.
[server] 
[server]     at createTSError (/node_modules/ts-node/src/index.ts:859:12)
[server]     at reportTSError (/node_modules/ts-node/src/index.ts:863:19)
[server]     at getOutput (/node_modules/ts-node/src/index.ts:1077:36)
[server]     at Object.compile (/node_modules/ts-node/src/index.ts:1433:41)
[server]     at Module.m._compile (/node_modules/ts-node/src/index.ts:1617:30)
[server]     at node:internal/modules/cjs/loader:1943:10
[server]     at Object.require.extensions.<computed> [as .ts] (/node_modules/ts-node/src/index.ts:1621:12)
[server]     at Module.load (node:internal/modules/cjs/loader:1533:32)
[server]     at Module._load (node:internal/modules/cjs/loader:1335:12)
[server]     at wrapModuleLoad (node:internal/modules/cjs/loader:255:19) {
[server]   diagnosticCodes: [ 2322, 2322 ]
[server] }
[worker] npm run dev:worker exited with code 1
--> Sending SIGTERM to other processes..
[server] npm run dev:server exited with code 1
```

**Environment (please complete the following information):**

- @vendure/core version: 3.6.3
- Nodejs version: 24.14.0
- Database (mysql/postgres etc):  postgres
- Operating System (Windows/macOS/Linux): macOS
- Browser (if applicable):
- Package manager (npm/yarn/pnpm): npm

**Configuration**

**Minimal reproduction**

**Workaround**
Adding this to package.json works:

```json
  "overrides": {
    "graphql": "16.13.2"
  }
```

**Additional context**
No problem on 3.6.2

--- a/package.json
+++ b/package.json
@@ -52,7 +52,7 @@
     "cross-env": "^7.0.3",
     "find": "^0.3.0",
     "gql.tada": "^1.8.10",
-    "graphql": "~16.11.0",
+    "graphql": "^16.11.0",
     "husky": "^4.3.0",
     "lerna": "^9.0.3",
     "lint-staged": "^10.5.4",

--- a/package.json
+++ b/package.json
@@ -76,6 +76,10 @@
     "@swc/core-darwin-arm64": "1.7.19",
     "@swc/core-linux-x64-gnu": "1.4.7"
   },
+  "//overrides": "Force a single graphql version across the monorepo install. Without this, transitive deps (e.g. @types/graphql-upload) keep a nested graphql@16.11.0 alongside the hoisted copy, which breaks TS type portability (TS2742) when building @vendure/core. Mirrors the ^16.11.0 dependency ranges. Relates to #4701.",
+  "overrides": {
+    "graphql": "^16.11.0"
+  },
   "trustedDependencies": [
     "husky",
     "better-sqlite3",

--- a/packages/admin-ui/package.json
+++ b/packages/admin-ui/package.json
@@ -60,7 +60,7 @@
         "chartist": "^1.3.0",
         "codejar": "^4.2.0",
         "dayjs": "^1.11.10",
-        "graphql": "~16.11.0",
+        "graphql": "^16.11.0",
         "just-extend": "^6.2.0",
         "messageformat": "2.3.0",
         "ngx-pagination": "^6.0.3",

--- a/packages/core/package.json
+++ b/packages/core/package.json
@@ -59,7 +59,7 @@
         "csv-parse": "^6.2.1",
         "express": "^5.1.0",
         "fs-extra": "^11.2.0",
-        "graphql": "~16.11.0",
+        "graphql": "^16.11.0",
         "graphql-fields": "^2.0.3",
         "graphql-scalars": "^1.24.2",
         "graphql-tag": "^2.12.6",

--- a/packages/dashboard/package.json
+++ b/packages/dashboard/package.json
@@ -107,7 +107,7 @@
         "fast-glob": "^3.3.2",
         "fs-extra": "^11.3.4",
         "gql.tada": "^1.8.13",
-        "graphql": "~16.11.0",
+        "graphql": "^16.11.0",
         "input-otp": "^1.4.2",
         "json-edit-react": "^1.23.1",
         "lucide-react": "^0.475.0",

--- a/packages/testing/package.json
+++ b/packages/testing/package.json
@@ -41,7 +41,7 @@
         "@vendure/common": "3.6.3",
         "faker": "^4.1.0",
         "form-data": "^4.0.0",
-        "graphql": "~16.11.0",
+        "graphql": "^16.11.0",
         "graphql-tag": "^2.12.6",
         "node-fetch": "^2.7.0",
         "sql.js": "1.14.1"

